spec/system/admin_customize_themes_spec.rb

```diff
@@ -160,5 +160,34 @@
       theme_translations_settings_editor.fill_in("Hello World in French")
       theme_translations_settings_editor.save
     end
+
+    # relates to https://meta.discourse.org/t/locale-mismatch-at-theme-translations/302879
```

Comment:
Probably better to leave that link in the commit message rather than in the code.

Reply:
Sure, the link is in the commit as well.

I'll remove the comment